README.md

```diff
@@ -26,7 +26,7 @@ Stable releases can be found on the [GitHub Releases page](https://github.com/ds
 npm install dso-toolkit --save-dev
 ```
 
-1. Add `node_modules/dso-toolkit/libs` and `node_modules` as [include paths](https://github.com/sass/node-sass#includepaths) to your SASS compiler/wrapper (See [#105](https://github.com/dso-toolkit/dso-toolkit/issues/105) for more information)
+1. Add `node_modules/dso-toolkit/libs` as [include paths](https://github.com/sass/node-sass#includepaths) to your SASS compiler/wrapper (See [#105](https://github.com/dso-toolkit/dso-toolkit/issues/105) for more information)
```

Comment:
@WiebeCnossen Volgens mij is de include path naar `node_modules` niet meer nodig omdat we `bootstrap` hebben opgenomen in `libs/`, kan jij dit bevestigen in jouw builds?

Reply:
Hmm, dat is nog niet zo makkelijk te zeggen, omdat we nu in de implementatie er gebruik van maken dat het in het pad staat.

Reply:
Ja, als ik dat in de implementatie aanpas, dan werkt het inderdaad zonder de `node_modules` include. Maar die ga ik in de implementatie wel weer terugzetten, want dat is wel zo handig.

Reply:
Ok, dank voor het testen